### PR TITLE
Add a --no-reload option to not reload py and jinja files in non-production mode

### PR DESCRIPTION
**Pull request by JackMc**
Also fixes a bug where the web.sh script used non-capitalized True

### CODE REVIEW

**Comment by JackMc** [comment]
The bug fixed here is making a lot of things rear their ugly heads...

**Comment by JackMc** [comment]
@deniszgonjanin @sonalsharma This should work for your purposes on Mac

**Comment by JackMc** [comment]
Ready for CR and tests are passing @wardi

File: datacats/scripts/web.sh [hunk 1]
```diff
@@ -18,16 +18,20 @@ fi
...
-	if [ "$2" = "true" ]; then
+	if [ "$2" = "True" ]; then
 		/scripts/adjust_devini.py "$(ip route get 8.8.8.8 | awk 'NR==1 {print $NF}')" "$port"
 	fi
 
 	# production
-	if [ "$1" = "true" ]; then
-		/usr/bin/apachectl -DFOREGROUND
-	else
+	if [ "$1" = "True" ]; then
```
Comment (wardi) [inline datacats/scripts/web.sh]: there's a real tab here! nooooooooo

File: datacats/scripts/web.sh [hunk 2]
```diff
@@ -18,16 +18,20 @@ fi
...
+		/usr/sbin/apachectl -DFOREGROUND
+        elif [ "$3" == "True" ]; then
```
Comment (wardi) [inline datacats/scripts/web.sh]: I mean.. there's spaces here ! noooooooooo

File: datacats/cli/install.py [hunk 3]
```diff
@@ -40,7 +40,8 @@ def install(environment, opts):
             'PORT': None,
             '--background': False,
             '--address': opts['--address'],
-            '--syslog': False})
+            '--syslog': False,
+            '--no-reload': False})
```
Comment (wardi) [inline datacats/cli/install.py]: nit picking here, but.. try to keep these alphabetical or chaos will descend on our beautiful little village and devour everyone.

**Comment by wardi** [comment]
Odd CI failure. wonder what's going on there

**Comment by JackMc** [comment]
I have no idea why it's failing there and not on the previous commit

**Comment by JackMc** [comment]
For some reason the permissions on the web.sh script got put to 655 instead of 755. Hence, it wasn't able to execute it.

File: datacats/cli/manage.py [hunk 4]
```diff
@@ -66,15 +67,16 @@ def reload_(environment, opts):
     """Reload environment source and configuration
 
 Usage:
-  datacats reload [-bp] [--syslog] [-s NAME] [--address=IP] [ENVIRONMENT [PORT]]
+  datacats reload [-b] [-p|--no-reload] [--syslog] [-s NAME] [--address=IP] [ENVIRONMENT [PORT]]
```
Comment (wardi) [inline datacats/cli/manage.py]: `datacats reload --no-reload` it's very zen.

Comment (JackMc) [inline datacats/cli/manage.py]: I try my best to provide you with the best philosophical insights in my code. :stuck_out_tongue: 

But yeah this naming sucks. Any ideas?

Comment (wardi) [inline datacats/cli/manage.py]: I'm thinking either:
1. `datacats restart` while quietly redirecting `reload` to the new command.
2. `datacats reload --no-watch` to say that we're not watching the files for changes

Comment (JackMc) [inline datacats/cli/manage.py]: I like # 2

Comment (wardi) [inline datacats/cli/manage.py]: dcdf5b42acb95fa3551faa23a2f1e9e4

**Pull request merged**